src/main/java/items/ConsumableStatusItem.java

```diff
@@ -0,0 +1,32 @@
+package items;
+
+
+/**
+ * An class representing a healing consumable item.
+ */
+public class ConsumableStatusItem extends Item implements Consumable{
+
+    private String statusType;
+
+    public ConsumableStatusItem(String name, String description, String statusType){
+        super(name, description);
+        this.statusType = statusType;
+    }
+
+    public String getStatusType(){
+        return this.statusType;
+    }
+
+    public void consume(Character consumer){
+        if (this.statusType == consumer.getStatus){
```

Comment:
what if you're affected by multiple status types

Reply:
honestly, not sure, i think we can just leave it at one status type at atime

Reply:
in that case, how do we want to handle status application on someone already affected by a different one. Will it not work or will it replace the old one.